packages/schema/lib/schemas/AuthInputFieldSchema.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+const FieldSchema = require('./FieldSchema');
+
+module.exports = makeSchema(
+  {
+    ...FieldSchema.schema,
+    id: '/AuthInputFieldSchema',
+    description:
+      'Field schema specialized for auth input fields (e.g., OAuth).',
```

Comment:
```suggestion
      'Field schema specialized for authentication fields.',
```

The examples/anti-examples would give the hint on the auth types.